pkg/docker/containers-view.jsx

```diff
@@ -129,12 +129,38 @@ var ContainerDetails = React.createClass({
     }
 });
 
+var ContainerProblems = React.createClass({
+    onItemClick: function (event) {
+        cockpit.jump(event.currentTarget.dataset.url);
+    },
+
+    render: function () {
+        var problem = this.props.problem;
+
+        var problem_cursors = [];
+        for (var i = 0; i < problem.length; i++) {
+            problem_cursors.push(<a data-url={problem[i][0]} className='list-group-item' onClick={this.onItemClick}>
+            <span className="pficon pficon-error-circle-o fa-lg"></span>
+            {problem[i][1]}
+            <i className="pull-right pficon fa fa-angle-right fa-lg"></i>
```

Comment:
It is up to you. I thought it would look nice and also I probably saw it somewhere else. But if you think it should not be there, no problem removing it. Let me know.

Reply:
I don't think we use it anywhere else, so it would be inconsistent with other places in Cockpit.
Thanks for getting rid of it!